README.md

```diff
@@ -5,219 +5,722 @@ MailerSend Python SDK
 
-subject = "Subject"
-text = "This is the text content"
-html = "<p>This is the HTML content</p>"
+recipients = [
+    'your@client.com',
+    'another@client.com',
+]
 
-my_mail = "owner@verified_domain.com"
-recipient_list = [ 'pamela@dundermifflin.com',
-'dwight@dunderfmifflin.com', 'jim@dundermifflin.com']
+mailer.setMailFrom('my-verified@mail.com', mail_body)
+mailer.setMailTo(recipients, mail_body)
+mailer.setSubject("Hello!", mail_body)
+mailer.setHTMLContent("This is the HTML content", mail_body)
+mailer.setPlaintextContent("This is the text content", mail_body)
 
-mailer.send(my_mail, recipient_list, subject, html, text, None, None)
+# using print() will also return status code and data
+mailer.send(mail_body)
 ```
 
-### Sending a basic email using template ID.
+<a name="cc_and_bcc"></a>
+## Sending an email with CC and BCC
+
+Send an email with CC and BCC.
+
+### Using just email addresses for CC and BCC
+
+```python
+import mailersend.emails.emails as emails
+
+mailer = emails.NewEmail()
 
-``` python
-import mailersend
+# define an empty dict to populate with mail values
+mail_body = {}
 
-mailer = mailersend.NewApiClient()
+recipients = [
+    'your@client.com',
+    'another@client.com
+]
+
+cc = [
+    'yetanother@client.com',
+]
 
-subject = "Subject"
-template_id = '351ndrxrx45zqx8k'
+bcc = [
+    'myboss@company.com',
+]
 
-my_mail = "owner@verified_domain.com"
-recipient_list = [ 'pamela@dundermifflin.com',
-'dwight@dunderfmifflin.com', 'jim@dundermifflin.com']
+mailer.setMailFrom('my-verified@mail.com', mail_body)
+mailer.setMailTo(recipients, mail_body)
+mailer.setSubject("Hello!", mail_body)
+mailer.setHTMLContent("This is the HTML content", mail_body)
+mailer.setPlaintextContent("This is the text content", mail_body)
+mailer.setCCRecipients(cc, mail_body)
+mailer.setBCCRecipients(bcc, mail_body)
 
-mailer.send(my_mail, recipient_list, subject, None, None, template_id, None)
+mailer.send(mail_body)
 ```
 
-### Sending an email with simple personalization.
+### Using email and name for CC and BCC
 
-``` python
-import mailersend
+```python
+import mailersend.emails.emails as emails
+
+mailer = emails.NewEmail()
 
-mailer = mailersend.NewApiClient()
+# define an empty dict to populate with mail values
+mail_body = {}
 
-subject = "Hello from {$company}"
-text = "This is the text content"
-html = "<p>This is the HTML content, {$name}</p>"
+recipients = [
+    'your@client.com',
+    'another@client.com
+]
 
-my_mail = "owner@verified_domain.com"
-recipient_list = [ 'pamela@dundermifflin.com',
-'dwight@dunderfmifflin.com', 'jim@dundermifflin.com']
-variables = [{
-    "email": "pamela@dundermifflin.com",
-    "substitutions": [{
-        "var": "name",
-        "value": "Pam",
+cc = [
+    {
+        "name": "Client1",
+        "email": "yetanother@client.com"
     },
     {
-        "var": "company",
-        "value": "Dunder Mifflin",
-    },]
-}]
+        "name": "Client2",
+        "email": "justanother@client.com"
```

Comment:
could we try and keep emails the same throughout our SDK's 

https://github.com/mailersend/mailersend-go

cc@client.com
bcc@client.com

etc